x-pack/test/security_solution_endpoint_api_int/apis/endpoint_artifacts/host_isolation_exceptions.ts

```diff
@@ -131,88 +131,105 @@ export default function ({ getService }: FtrProviderContext) {
+            const body = apiCall.getBody();
+
+            body.entries[0].field = 'some.invalid.field';
+
+            await supertest[apiCall.method](apiCall.path)
+              .set('kbn-xsrf', 'true')
+              .send(body)
+              .expect(400)
+              .expect(anEndpointArtifactError)
+              .expect(anErrorMessageWith(/expected value to equal \[destination.ip\]/));
+          });
+
+          it(`[${apiCall.method}] if more than one entry`, async () => {
+            const body = apiCall.getBody();
+
+            body.entries.push({ ...body.entries[0] });
+
+            await supertest[apiCall.method](apiCall.path)
+              .set('kbn-xsrf', 'true')
+              .send(body)
+              .expect(400)
+              .expect(anEndpointArtifactError)
+              .expect(anErrorMessageWith(/\[entries\]: array size is \[2\]/));
+          });
+
+          it(`[${apiCall.method}] if an invalid ip is used`, async () => {
+            const body = apiCall.getBody();
+
+            body.entries = [
+              {
+                field: 'destination.ip',
+                operator: 'included',
+                type: 'match',
+                value: 'not.an.ip',
+              },
+            ];
+
+            await supertest[apiCall.method](apiCall.path)
+              .set('kbn-xsrf', 'true')
+              .send(body)
+              .expect(400)
+              .expect(anEndpointArtifactError)
+              .expect(anErrorMessageWith(/invalid ip/));
+          });
+
+          it(`[${apiCall.method}] if all OSs for os_types are not included`, async () => {
+            const body = apiCall.getBody();
+
+            body.os_types = ['linux', 'windows'];
+
+            await supertest[apiCall.method](apiCall.path)
+              .set('kbn-xsrf', 'true')
+              .send(body)
+              .expect(400)
+              .expect(anEndpointArtifactError)
+              .expect(anErrorMessageWith(/\[osTypes\]: array size is \[2\]/));
+          });
+
+          it(`[${apiCall.method}] if policy id is invalid`, async () => {
+            const body = apiCall.getBody();
+
+            body.tags = [`${BY_POLICY_ARTIFACT_TAG_PREFIX}123`];
+
+            await supertest[apiCall.method](apiCall.path)
+              .set('kbn-xsrf', 'true')
+              .send(body)
+              .expect(400)
+              .expect(anEndpointArtifactError)
+              .expect(anErrorMessageWith(/invalid policy ids/));
+          });
+        }
+      });
 
-        it(`should error on [${apiCall.method}] if policy id is invalid`, async () => {
-          const body = apiCall.getBody();
+      describe('should work on ', () => {
+        for (const apiCall of apiCalls) {
+          it(`[${apiCall.method}] if entry is valid`, async () => {
+            const body = apiCall.getBody();
 
-          body.tags = [`${BY_POLICY_ARTIFACT_TAG_PREFIX}123`];
+            await supertest[apiCall.method](apiCall.path)
```

Comment:
You are still leaving data behind on the create. I think you need to capture the return item created, then delete it after you do the assertion (similar to what we do with the `existingExceptionData` that gets created in the `beforeEach()`

Proposal: add the create case to the Endpoint UI FTR tests suite instead if you want to ensure it is successful.